central/graphql/resolvers/plotted_node_vulnerabilities.go

```diff
@@ -0,0 +1,61 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+func init() {
+	schema := getBuilder()
+	utils.Must(
+		schema.AddType("PlottedNodeVulnerabilities", []string{
```

Comment:
This is a top-level type therefore, this can be invoked directly. However, the resolver assumes that the cve type `NODE_CVE` is always provided, which is not true.

Reply:
Yeah that is right, instead of adding `CVEType = NODE_CVE` filter on every `PlottedNodeVulnerabilities` sub-resolver call, I added it to the init function `newNodeVulnerabilitiesResolver` itself.